docs/content/preview/yugabyte-voyager/reference/non-superuser.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Import data without a superuser
+linktitle: Non-superuser data import
+description: Importing data without a superuser in YugabyteDB
+menu:
+  preview_yugabyte-voyager:
+    identifier: yb-voyager-superuser
+    parent: yb-voyager-misc
+    weight: 120
+type: docs
+---
+
+This document provides a step-by-step guide to importing data into YugabyteDB without using a superuser account. It outlines why superuser access is required in some cases, how to work around it, and the precise steps to follow to import data cleanly.
+
+## Why do you need a Superuser?
+
+Superuser access is primarily needed to:
+
+- Set `session_replication_role` to `replica` (used to temporarily disable triggers and foreign keys during import).
+
+- Create extensions like pgcrypto.
+
+- Create extensions like [postgres_fdw](https://www.postgresql.org/docs/current/postgres-fdw.html) or [dblink](https://www.postgresql.org/docs/current/dblink.html).
+
+  - These extensions define foreign data wrappers and access mechanisms that interact with external databases.
+  - They register C functions and system-level handlers — only superusers can install them.
+
+- Create custom base types.
+
+  - Base types (`CREATE TYPE ...`) require defining low-level behaviors (such as INPUT and OUTPUT functions).
+  - These are internally linked to system operations and can affect core engine behavior.
+
+- Use of `LANGUAGE internal` in functions.
+
+  - Functions using `LANGUAGE internal` map directly to compiled C functions (like `boolin`).
+
+- Any other unknown cases.
+
+## Import without a Superuser
+
+Starting from YugabyteDB v2025.1 (which is based on PostgreSQL 15), you can grant users the ability to `set session_replication_role` without making them superusers by directly granting the permission as follows:
```

Comment:
this section is a bit confusing. 
- Can this be placed under `Grant required permissions` as a note? 
- and in `Import Data Errors` section, re-iterate/highlight that for 2025.1 onwards, they can just run the GRANT session_replication_role , there is no need to manually disable/re-enable Triggers and FKs? cc: @ShivanshGahlot